### DISCUSSION TITLE
SOLVED : How to configure a public/provate key pair for this SSH addon ?

### DISCUSSION THREAD

**Discussion by HuguesD** [hassio-addons/addon-ssh#356] [General]
Hi,

I am a newbie and have just configured a home assistant server on a raspberry PI4. I have added this SSH addon to it and I can SSH to it via the integrated web browser of home assistant.
I have putty on windows and would like to SSH to it remotely. But I want to install a public/private key pair (I have generated the key pair via puttygen).
How do I configure the private/public key pair in this SSH module ? Is there any dcumentation with step by step commands to run?

thanks

**Comment by frenck** [comment]
If you go to the add-on in Home Assistant, you'll find a "Documentation" tab for that add-on.
It contains documentation on the configuration options and an configuration example:

![image](https://user-images.githubusercontent.com/195327/146673747-d68706fa-bc6c-4e0f-b366-bf7d8685d1fb.png)

**Reply by HuguesD** [reply]
Thanks. I have added the public key in this config file and i have wriiten "enabled" in the network section. However Putty returns "connection refused" (in putty I have configured the IP addres, port 22 and I have selected the private key in the SSL section). See screensho
![Sans titre 1](https://user-images.githubusercontent.com/13796161/146673970-855d9275-46e0-47ac-a6ee-2b71f3f3f337.jpg)
t

**Reply by frenck** [reply]
![image](https://user-images.githubusercontent.com/195327/146674018-c0bce788-f72b-492f-b4d8-161a48f56979.png)

SSH is disabled, as there is no port assigned to the host. Update `enabled` -> `22`, save it and restart the add-on.

Additionally, by the looks of it, you are in the wrong repository. The configuration shown is not this add-on.

This is the community SSH add-on, you are showing configuration of the official core add-on. (So you are in the wrong place).

**Reply by HuguesD** [reply]
Thanks it works now. Eventhough it is not the same addon

**Reply by HuguesD** [reply]
closed issue
